portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -0,0 +1,38 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+@WebServlet("/login")
+public class LoginServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("text/html");
+
+    UserService userService = UserServiceFactory.getUserService();
+    if (userService.isUserLoggedIn()) {
+      String userEmail = userService.getCurrentUser().getEmail();
+      String urlToRedirectToAfterUserLogsOut = "/comments.html";
+      String logoutUrl = userService.createLogoutURL(urlToRedirectToAfterUserLogsOut);
+      
+      String json = "{\"login\": \"true\", \"url\": \"" + logoutUrl + "\", \"email\": \"" + userEmail + "\"}";
+      response.getWriter().println(json);
+    } else {
+      String urlToRedirectToAfterUserLogsIn = "/comments.html";
+      String loginUrl = userService.createLoginURL(urlToRedirectToAfterUserLogsIn);
+      
+      String json = "{\"login\": \"false\", \"url\": \"" + loginUrl + "\", \"email\": \"" + "false" + "\"}";
```

Comment:
nit: It may be cleaner to create a Java object and convert it to json. It may make it easier to see the following:

not nit: noticed that email here is set to the string "false". Is that intended? Would it be cleaner to not set it at all? Or set it to empty?

Reply:
Done: made a separate object and initialized the email to empty string